devops/kubernetes/charts/origin-experimental/templates/dshop-backend-mainnet.statefulset.yaml

```diff
@@ -27,6 +27,8 @@ spec:
         image: "{{ .Values.containerRegistry }}/{{ .Release.Namespace }}/{{ .Values.dshopBackendMainnetImage }}:{{ .Values.dshopBackendMainnetImageTag }}"
         imagePullPolicy: Always
         env:
+          - name: ENVIRONMENT
```

Comment:
@franckc   Sorry for belated feedback but was just thinking about when to use env vars and how to reduce them.  Why use this at all?  Why not just use `SENTRY_DSN` to figure out if errors should be sent?